include/s3select_functions.h

```diff
@@ -81,16 +85,20 @@ class s3select_functions
     {"dateadd", s3select_func_En_t::DATE_ADD},
     {"datediff", s3select_func_En_t::DATE_DIFF},
     {"utcnow", s3select_func_En_t::UTCNOW},
-    {"charlength", s3select_func_En_t::LENGTH},
-    {"characterlength", s3select_func_En_t::LENGTH},
+    {"character_length", s3select_func_En_t::LENGTH},
+    {"char_length", s3select_func_En_t::LENGTH},
     {"lower", s3select_func_En_t::LOWER},
     {"upper", s3select_func_En_t::UPPER},
     {"nullif", s3select_func_En_t::NULLIF},
     {"#between#", s3select_func_En_t::BETWEEN},
     {"#is_null#", s3select_func_En_t::IS_NULL},
     {"#in_predicate#", s3select_func_En_t::IN},
     {"#like_predicate#", s3select_func_En_t::LIKE},
-    {"version", s3select_func_En_t::VERSION}
+    {"version", s3select_func_En_t::VERSION},
+    {"#when-than#", s3select_func_En_t::WHEN_THAN},
+    {"#case-when-else#", s3select_func_En_t::CASE_WHEN_ELSE},
+    {"coalesce", s3select_func_En_t::COALESCE},
+    {"cast", s3select_func_En_t::CAST}
```

Comment:
not correct.
cast is general operator , it invoke different types of converting functions
to_string is missing (int,double,timestamp already exist)